exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/union/UnionAllRecordBatch.java

```diff
@@ -409,6 +426,22 @@ public void remove() {
   public void close() {
     super.close();
     updateBatchMemoryManagerStats();
+
+    logger.debug("BATCH_STATS, incoming aggregate left: batch count : {}, avg bytes : {},  avg row bytes : {}, record count : {}",
```

Comment:
This does not have to be under isDebugEnabled flag

Reply:
The reason for putting it is so that for each log statement internally it doesn't check for debug level being enabled. Having single outer check will help here.